.github/workflows/verify_chromatic.yml

```diff
@@ -56,3 +57,28 @@ jobs:
           projectToken: chpt_dab72dc0f97d55b
           storybookBuildDir: dist-storybook
           onlyChanged: true
+          externals: |
+            packages/ui/**/*.css
+            packages/ui/css/**
```

Comment:
Should add these to the workflow trigger as well

Reply:
You're right. I changed `packages/ui/src/**` to `packages/ui/**`